fiat-roles/src/main/java/com/netflix/spinnaker/fiat/roles/google/GoogleDirectoryUserRolesProvider.java

```diff
@@ -167,6 +174,10 @@ private Directory getDirectoryService() {
         .build();
   }
 
+  private static Role toRole(Group g) {
```

Comment:
You could factor `toRole` out into a utility function that also takes the `source` as a parameter as well rather than having one per provider.